backstop_data/engine_scripts/puppet/search/actions/common.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/**
+ * Performs a contact search, then selects the checkboxes with the given ids
+ *
+ * The search params are defined so that the smallest useful recordset is returned
+ * as a way to improve performance
+ *
+ * @param {CrmPage} page
+ * @param {Array} checkboxIds
+ */
+module.exports = async (page, checkboxIds=['3']) => {
```

Comment:
Why there is a default id `3`?

Reply:
It's just the default contact's checkbox that gets selected by 99% of the search action scripts